bears/python/YapfBear.py

```diff
@@ -0,0 +1,135 @@
+            expression can't fit in a single line. Applies to all kinds of
+            brackets, including function definitions and calls.
+        :param indent_dictionary_value:
+            Indent the dictionary value if it cannot fit on the same line as
+            the dictionary key.
+        :param join_multiple_lines:
+            Join short lines into one line.
+        :param spaces_around_power_operator:
+            Set to ``True`` to prefer using spaces around ``**``.
+        :param spaces_before_comment:
+            The number of spaces required before a trailing comment.
+        :param space_between_ending_comma_and_closing_bracket:
+            Inserts a space between the ending comma and closing bracket of a
+            list, etc.
+        :param split_arguments_when_comma_terminated:
+            Splits before arguments if the argument list is terminated by a
+            comma.
+        :param split_before_bitwise_operator:
+            Set to ``True`` to prefer splitting before ``&``, ``|`` or ``^``
+            rather than after.
+        :param split_before_first_argument:
+            If an argument / parameter list is going to be split, then split
+            before the first argument.
+        :param split_before_logical_operator:
+            Set to ``True`` to prefer splitting before ``and`` or ``or`` rather
+            than after.
+        :param split_before_named_assigns:
+            Splits named assignments into individual lines.
+        :param use_tabs:
+            Use the Tab character for indentation.
+        :param based_on_style:
+            The formatting style to be used as reference.
+        """
+
+        options = """
+        [style]
+        indent_width = {tab_width}
+        column_limit = {max_line_length}
+        allow_multiline_lambdas = {allow_multiline_lambdas}
+        continuation_indent_width = {continuation_tab_width}
+        dedent_closing_brackets = {dedent_closing_brackets}
+        indent_dictionary_value = {indent_dictionary_value}
+        join_multiple_lines = {join_multiple_lines}
+        spaces_around_power_operator = {spaces_around_power_operator}
+        spaces_before_comment = {spaces_before_comment}
+        split_before_bitwise_operator = {split_before_bitwise_operator}
+        split_before_first_argument = {split_before_first_argument}
+        split_before_logical_operator = {split_before_logical_operator}
+        split_before_named_assigns = {split_before_named_assigns}
+        based_on_style = {based_on_style}
+        """
+        options += ('blank_line_before_nested_class_or_def = '
+                    '{blank_line_before_nested_class_or_def}\n'
+                    'space_between_ending_comma_and_closing_bracket = '
+                    '{space_between_ending_comma_and_closing_bracket}\n'
+                    'split_arguments_when_comma_terminated = '
+                    '{split_arguments_when_comma_terminated}\n')
```

Comment:
maybe the names can be shortened anyway? Does https://github.com/coala-analyzer/coala-bears/pull/578/files/26fb161a0e036fc18be15be6f7f0b034d9bbf751#r71331494 help?